hotels.css

```
@@ -0,0 +1,132 @@
+@import url("https://fonts.googleapis.com/css2?family=Dancing+Script&family=Pacifico&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+/* CONTAINER header */
+.container-header {
+  /* background-color: blue; */
+  background-image: url(images/playa-mar2.png);
+  padding: 48px 48px 28px;
+  height: 300px;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
```

Te recomiendo agregar esta linea para darle mejor visualización al fondo:

```suggestion
    background-position: center;
}
```

Si te gusta, haz el cambio localmente y le haces push a la rama otra vez.

---

Buena tarde  profe se realizan los cambios sugeridos queda pendiente ajustar el total del height de la pagina para q no quede tanto espacio blanco al final pero ese lo realizo para el siguiente PR ahora que comience a ajustar las funciones de los filtros